vue/src/components/templates/MainPage.vue

```diff
@@ -6,7 +6,8 @@
                 <cas-header
                     :hosted-by-text="hostedByText"
                     :title-text="titleText"
-                    :logo-url="logoUrl">
```

Comment:
add linter rule to move the > to the next line, for the same reason of comma-dangle

Reply:
will be fixed with eslint ticket